MaxMind.Db.Test/Helper/NoNetworkTypeHolder.cs

```diff
@@ -0,0 +1,135 @@
+﻿#region
```

Comment:
This is just a class used in the deserialization tests. It is the same as `TypeHolder` except without the new network fields. I did this so that I didn't have to make significant changes to the database enumeration tests.